src/command_modules/azure-cli-sql/setup.py

```diff
@@ -24,7 +24,8 @@
 
 DEPENDENCIES = [
     'azure-cli-core',
-    'azure-mgmt-sql==0.3.0'
+    'azure-mgmt-sql==0.3.0',
+    'six'
```

Comment:
I thought that since I am directly depending on six it should be called out as an explicit dependency. This way we don't break if theoretically azure-cli-core removed the dependency on six. Do you still want me to remove it? (I'm not pushing back - just explaining my thought process)